File: src/java/CdbWebPortal/src/java/gov/anl/aps/cdb/common/objects/Image.java
/*
 * Copyright (c) UChicago Argonne, LLC. All rights reserved.
 * See LICENSE file.
 */
package gov.anl.aps.cdb.common.objects;

/**
 *
 * @author djarosz
 */
public class Image extends CdbObject {
    String imageUrl; 
    String thumbnailUrl; 

    public String getImageUrl() {
        return imageUrl;
    }

    public String getThumbnailUrl() {
        return thumbnailUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public void setThumbnailUrl(String thumbnailUrl) {
        this.thumbnailUrl = thumbnailUrl;
    }
    
    
}

File: tools/developer_tools/cdb_plugins/plugins/pdmLink/python/objects/image.py
#!/usr/bin/env python

"""
Copyright (c) UChicago Argonne, LLC. All rights reserved.
See LICENSE file.
"""


from cdb.common.objects.cdbObject import CdbObject


class Image(CdbObject):

    DEFAULT_KEY_LIST = ['imageUrl', 'thumbnailUrl']

    def __init__(self, dict):
        CdbObject.__init__(self, dict)

